core/classes/Core/Input.php

```diff
@@ -129,7 +129,7 @@ public static function createTinyEditor(Language $language, string $name, ?strin
               external_plugins: {
                 'spoiler': '" . URL::build('/core/assets/plugins/tinymce_spoiler/plugin.min.js') . "',
               },
-              toolbar: 'undo redo | bold italic underline strikethrough formatselect fontsizeselect forecolor backcolor ltr rtl emoticons | alignleft aligncenter alignright alignjustify | codesample code hr image link numlist bullist | spoiler-add spoiler-remove',
+              toolbar: 'undo redo | bold italic underline strikethrough formatselect fontsizeselect forecolor backcolor ltr rtl emoticons | alignleft aligncenter alignright alignjustify | codesample " . ($admin ? "code" : "") . " hr image link numlist bullist | spoiler-add spoiler-remove',
```

Comment:
I guess the extra space if admin is disabled (it'll render as `codesample  hr` won't matter?

Reply:
Didn't matter when I tested at least